actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -416,6 +383,14 @@ def values
       to_enum(:each_value).to_a
     end
 
+    # Returns +true+ if the given key is present in the parameters.
+    def include?(key)
```

Comment:
Let's implement `has_key?` and alias using that. It will be more consistent with https://github.com/rails/rails/pull/44866